### DISCUSSION TITLE
Delay/latency

### DISCUSSION THREAD

**Discussion by Infinitay** [younesaassila/ttv-lol-pro#122] [Q&A] [answered]
I would imagine there is latency added when a stream first loads when proxying the playlist from a third-party. Could I still use  FFZ to reload the player and the stream to reduce the delay like you would a stream without the extension? Also, if I were to add a website to my whitelist to not use the extension because I'm subbed to them, would it still add latency?

Oh and a bonus question, even if I run the extension, does it still _technically "watch" the ads_ and gives streamers the ad revenue?

**Comment by younesaassila** [comment, marked as answer]
> I would imagine there is latency added when a stream first loads when proxying the playlist from a third-party. Could I still use FFZ to reload the player and the stream to reduce the delay like you would a stream without the extension?

You can use FFZ but I doubt it would make any difference since the latency mainly comes from getting the stream from ad-free countries (which might be far away from you).

> Also, if I were to add a website to my whitelist to not use the extension because I'm subbed to them, would it still add latency?

No, because the stream will be fetched from the nearest Twitch server location to you.

> Oh and a bonus question, even if I run the extension, does it still technically "watch" the ads and gives streamers the ad revenue?

No, by proxying the stream to an ad-free country, the extension is basically telling Twitch "I can't watch ads since I'm from _insert ad-free country_" so Twitch doesn't bother delivering ads to you. This in turns means the streamer doesn't earn any revenue from you watching their stream. If you want to support them, either whitelist their channel or sub/donate bits to them.
